<launch>

    <param name="/localization_marker_server/marker/name" value="object" type="string" />
    <param name="/localization_marker_server/marker/description" value="Align Me!" type="string" />
    <param name="/localization_marker_server/marker/type" value="MESH" type="string" />
    <param name="/localization_marker_server/marker/mesh_resource" value="package://wpi_drc_sim/meshes/valve_models/logitech_driving_force_pro_wheel.dae" type="string" />
    <param name="/localization_marker_server/marker/defaultPose/position/x" value="0" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/position/y" value="0" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/position/z" value="0" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/orientation/w" value=".5" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/orientation/x" value=".5" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/orientation/y" value="-.5" type="double" />
    <param name="/localization_marker_server/marker/defaultPose/orientation/z" value=".5" type="double" />
    <param name="/localization_marker_server/marker/radius" value=".28" type="double" />
    <param name="/localization_marker_server/marker/width" value=".025" type="double" />
    <param name="/localization_marker_server/marker/height" value=".025" type="double" />
    <param name="/localization_marker_server/marker/scale" value=".025" type="double" />
    <param name="/localization_marker_server/marker/color/r" value="1" type="double" />
    <param name="/localization_marker_server/marker/color/g" value="0" type="double" />
    <param name="/localization_marker_server/marker/color/b" value="0" type="double" />
    <param name="/localization_marker_server/marker/color/a" value="1" type="double" />

    <param name="/localization_marker_server/bounding_box/show" value="true" type="bool" />
    <param name="/localization_marker_server/bounding_box/type" value="SPHERE" type="string" />
    <param name="/localization_marker_server/bounding_box/radius" value=".18" type="double" />
    <param name="/localization_marker_server/bounding_box/x" value=".3" type="double" />
    <param name="/localization_marker_server/bounding_box/y" value=".3" type="double" />
    <param name="/localization_marker_server/bounding_box/z" value=".3" type="double" />
    <param name="/localization_marker_server/bounding_box/color/r" value="0" type="double" />
    <param name="/localization_marker_server/bounding_box/color/g" value="0" type="double" />
    <param name="/localization_marker_server/bounding_box/color/b" value="1" type="double" />
    <param name="/localization_marker_server/bounding_box/color/a" value=".2" type="double" />

    <param name="/localization_marker_server/control/translation/x" value="true" type="bool" />
    <param name="/localization_marker_server/control/translation/y" value="true" type="bool" />
    <param name="/localization_marker_server/control/translation/z" value="true" type="bool" />
    <param name="/localization_marker_server/control/rotation/x" value="true" type="bool" />
    <param name="/localization_marker_server/control/rotation/y" value="true" type="bool" />
    <param name="/localization_marker_server/control/rotation/z" value="true" type="bool" />
    <param name="/localization_marker_server/control/menu" value="true" type="bool" />

    <param name="/localization_marker_server/target_cloud/topic" value="/kinect_head/depth_registered/points" type="string" />
    <param name="/localization_marker_server/target_cloud/voxel" value="false" type="bool" />
    <param name="/localization_marker_server/input_cloud/resource" value="/" type="string" />
    <param name="/localization_marker_server/input_cloud/discretization" value="100" type="int" />

    <param name="/localization_marker_server/icp/maxCoorespondanceDistance" value=".25" type="double" />
    <param name="/localization_marker_server/icp/maxInterations" value="500" type="int" />

    <param name="/localization_marker_server/hasChanged" value="false" type="bool" />

    <param name="/localization_marker_server/debug/input_cloud/pre_ICP/show" value="true" type="bool" />
    <param name="/localization_marker_server/debug/input_cloud/pre_ICP/topic" value="/localization_marker_server/debug/input_cloud/pre_icp" type="string" />
    <param name="/localization_marker_server/debug/input_cloud/post_ICP/show" value="true" type="bool" />
    <param name="/localization_marker_server/debug/input_cloud/post_ICP/topic" value="/localization_marker_server/debug/input_cloud/post_icp" type="string" />



    <node name="localization_marker_server" pkg="Localization_Markers" type="localization_marker_server" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find Localization_Markers)/rviz/localization_markers_pr2.vcg" />

</launch>
